### PR TITLE
🌱 [codebase-cleanup-2] docs: plan the second reliability cleanup series [step 1/17]

### PR DESCRIPTION
**Pull request by AlexDochioiu**
## Complexity

🌱 trivial — documentation only: a plan and tracker under `.plan/active/codebase-cleanup-2/`. No production, test, or generated code changes.

## What

Raises the second periodic reliability and maintainability cleanup series as a 17-step plan:

- **Bridge:** `forManifest` composition on the managed-runtime install and provision services plus one `ManagedRuntimeColdStartService` in `sesori_plugin_runtime` (replacing seven descriptor copies and two cold-start blocks); one connection view tracker (replacing the session and project trackers); the warm-up and glossary listener pairs merged into one owner per concern (D3); in-file duplicate loops folded in the worktree service and the Codex plugin; 53 log sites that interpolate caught errors moved to logger arguments.
- **Compatibility:** retire every marker dated `<= v1.6.0` whose peer is a released Sesori surface, under a new baseline decision (D1, default `>= v1.6.0`), with per-marker peer verification as in the first series.
- **Client:** eight identical Flutter platform adapters shared once in `module_app_ui` (D2); cubit composition for the session-detail, project-list, session-list, and new-session screens shared between the mobile and desktop shells through composition functions in `module_core/lib/src/di/`; one optimistic rename tracker and one model-variant derivation in `module_core` (the `isAvailable` divergence between New Session and the session screen is recorded as decision D7); auth client folds; a desktop attention reconciler with far fewer mutable parts, gated on the `desktop-app` plan retiring (D6).
- **Tests, tooling, docs:** identical fakes and repeated arrange blocks consolidated per workspace; unused dependencies, four ARB keys, and six dead symbols removed; historical documents outside `.plan/` deleted (D4); regression reconciliation and matrix run.

The plan also records a **Declined Candidates** table (exhaustive SSE ignore-lists, dispose-fence boilerplate, ACP per-adapter options services, core lifecycle owners, runtime manifests, `no_slop_linter` parity, the asymmetric session-options refresh listeners, the twin session-activity-analytics owners) so later sessions do not re-audit them without new evidence.

## Why

The first series retired on 2026-08-25. Since then ~79k lines landed (desktop shell, `module_app_ui`, sub-agent sessions, plugin warm-ups, session options announcements, seven managed-runtime descriptors) and duplication grew back in five concrete clusters, the test corpus (285k lines) now exceeds the production corpus (215k), and 81 compatibility markers still target a July baseline. The owner asked for another cleanup iteration focused on duplication, unwanted complexity, and low-value edge-case code.

## Risk and test focus

None for this PR. For the series, the risk profile is recorded per step: wire-contract change only in Step 7 (D1), moved production classes in Steps 2, 3, 4, 8, 9, 10, 11 (architecture-implementation review required), everything else behavior-preserving. The highest coverage level is L3 with the plugin and platform matrix recorded in `PLAN.md`.

## Expected result

No user-visible or database change. Reviewers should see a plan whose every step cites file paths and line ranges from `480d82f090`, a tracker that holds only decisions, principles, fixed titles, and the cleanup ledger (live status stays on GitHub; per-step evidence goes in `steps/step-NN.md`), and seven open decisions with defaults so execution never blocks.

## Decisions requested from the owner

D1 compat baseline (default `>= v1.6.0`), D2 shared adapters in `module_app_ui`, D3 merged listener pairs, D4 historical doc deletion, D5 reverse the first series' "install composition stays local", D6 Step 11 timing, D7 variant availability on the session screen. Silence keeps the defaults.

## Verification

- Documentation-only change: no Dart suites run. `git diff --check` clean.
- Evidence behind every step was verified against the current tree by grep or by reading the code; mechanical scans (normalized duplicate-block detection, dead top-level symbol census, pubspec dependency census, ARB key census, fake-body census) were run over every non-generated Dart file and cross-checked by hand.
- `dart analyze --fatal-infos` baseline for `bridge/app` (including `tool/benchmarks`) is clean at `480d82f090`.
- Architecture plan review (`architecture-plan-review` sub-agent): the draft was rejected with nine blocking findings (naming and collaborators of the cold-start helper, eager composition ownership, listener and adapter rules that also live in the reviewer skills, an asymmetric listener pair, empty-id normalization, `GetIt` in cubits, the twin analytics owners, the rename tracker's home). All nine were applied directly without a second review, per repository policy, so the plan is not described as reviewer-approved. Details are in the `## Plan Review` section of `TRACKER.md`.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

---
## Summary by cubic
Adds the second reliability cleanup series as a 17-step plan and tracker under `.plan/active/codebase-cleanup-2/`. Documentation only — no production, test, or generated code changes; the plan targets duplication that grew back since the first series retired in August.

**What the plan covers**

- Shares managed-runtime install/provision composition and the budgeted cold start across seven plugin descriptors in `sesori_plugin_runtime`.
- Replaces the session/project view trackers and two paired bridge listeners with single owners.
- Retires compatibility markers dated `≤ v1.6.0` with a released Sesori peer under new baseline decision D1.
- Moves eight identical Flutter platform adapters into `module_app_ui` and shares screen composition between the mobile and desktop shells.
- Consolidates repeated test fakes and arrange blocks; prunes unused dependencies, four ARB keys, and six dead symbols.
- Records declined candidates so later sessions do not re-audit them without new evidence.

**Decisions**

- Seven decisions (D1–D7) each have a default so execution never blocks: compat baseline `≥ v1.6.0`, shared adapters in `module_app_ui`, merged listener pairs, historical doc deletion, shared install composition, desktop attention timing, variant availability alignment.
- The architecture plan review's nine blocking findings were applied directly; the plan is not marked reviewer-approved.

<sup>Written for commit c7f35a5c7936ac3cc1b9ed7399b7b81436c8253a. Summary will update on new commits.</sup>

<a href="https://cubic.dev/pr/sesori-ai/sesori_apps_monorepo/pull/1296?utm_source=github" target="_blank" rel="noopener noreferrer" data-no-image-dialog="true"><picture><source media="(prefers-color-scheme: dark)" srcset="https://www.cubic.dev/buttons/review-in-cubic-dark.svg"><source media="(prefers-color-scheme: light)" srcset="https://www.cubic.dev/buttons/review-in-cubic-light.svg">Review in cubic</picture></a>

### CODE REVIEW

File: .plan/active/codebase-cleanup-2/PLAN.md [hunk 1]
```diff
@@ -0,0 +1,819 @@
...
+## Delivery Plan
+
+Fixed titles, complexity, and order. Targets are changed lines (additions plus
+deletions, generated code and tests included). Steps whose files do not
+overlap run in parallel (the owner previously asked for up to about five open
+PRs); the serialization column names the only hard ordering.
+
+| Step | Exact PR title | Target | After |
+|---|---|---:|---|
+| 1/17 | `🌱 [codebase-cleanup-2] docs: plan the second reliability cleanup series [step 1/17]` | 900–1,200 | — |
```
[the remark this answers is not in the document]
Comment (AlexDochioiu) [reply .plan/active/codebase-cleanup-2/PLAN.md:231]: [Sesori reply] Not changing this. The Target column is defined in the Delivery Plan preamble as "changed lines (additions plus deletions, generated code and tests included)" per PR, so the metric for Step 1 is the whole PR, not PLAN.md alone. This PR adds 819 + 185 = 1,004 lines, inside the declared 900–1,200 range. The plan is explicit that targets are estimates for honesty rather than acceptance criteria; per-step evidence files (steps/step-NN.md) record actuals for later steps.

**Comment by AlexDochioiu** [comment]
[Sesori reply] Addressed the single cubic finding inline: the Step 1 size target is PR-wide by the plan's own definition of the Target column, and the PR's actual changed-line count (1,004 across PLAN.md and TRACKER.md) is inside the declared 900–1,200 range, so no change was made.

**Comment by AlexDochioiu** [comment]
Superseded by #1295 at the owner’s request. The consolidated periodic-cleanup plan includes the useful work from this PR and a CONSOLIDATION.md ledger mapping every source step to its adopted, completed, or deferred disposition. Variant consolidation is covered by merged #1294; the plan also includes the requested repository-wide documentation audit and simplification of every regression guide.
